packages/@markuplint/rule-textlint/src/index.ts

```diff
@@ -1,33 +1,80 @@
 import { Result, createRule } from '@markuplint/ml-core';
-import { TextlintKernel } from '@textlint/kernel';
+import { TextlintKernel, TextlintResult } from '@textlint/kernel';
+import { TextlintKernelOptions } from '@textlint/kernel/lib/textlint-kernel-interface';
 
-// @ts-ignore / This has not types
+// @ts-ignore / This has no types
 import TextlintPluginHTML from 'textlint-plugin-html';
 
-const kernel = new TextlintKernel();
+import path from 'path';
 
-type Option = Partial<Parameters<TextlintKernel['lintText']>[1] & {}>;
+type Option = Partial<TextlintKernelOptions> | true;
+
+const isNode = typeof process !== 'undefined' && !!process.versions.node;
+
+let kernel: TextlintKernel;
+let textlint: typeof import('textlint');
+let _config: typeof import('textlint/lib/config/config');
 
 export default createRule<boolean, Option>({
 	defaultLevel: 'warning',
 	name: 'textlint',
 	defaultValue: true,
-	defaultOptions: {},
+	defaultOptions: isNode || {},
 	async verify(document, translate, config) {
 		const reports: Result[] = [];
 		const html = document.toString();
 		const option = config.option;
-		const textlintResult = await kernel.lintText(html, {
-			...option,
-			ext: '.html',
-			plugins: [
-				...(config.option.plugins || []),
-				{
-					pluginId: 'html',
-					plugin: TextlintPluginHTML,
-				},
-			],
-		});
+		let textlintResult: TextlintResult;
+		if (typeof option === 'object') {
+			if (!kernel) {
+				kernel = new TextlintKernel();
+			}
+
+			textlintResult = await kernel.lintText(html, {
+				...option,
+				ext: '.html',
+				plugins: [
+					...((config.option as TextlintKernelOptions).plugins || []),
+					{
+						pluginId: 'html',
+						plugin: TextlintPluginHTML,
+					},
+				],
+			});
+		} else if (option === true) {
+			if (!isNode) {
+				// eslint-disable-next-line no-console
+				console.error(
```

Comment:
Should we `throw new Error` here?

Reply:
@JounQin 

I hope it fetches the path if `isNode` is false.
In short, please use the `fetch` API.

Reply:
I'm not sure to understand, what should we `fetch`? `textlintrc` files could be different, and for file `a`, the `textlint` config can be at `..`, `../..`, etc, it is beyond the responsibility of this package IMO.

Reply:
@JounQin 

Oh right, This is loading automatically, I'm sorry.

Reply:
@JounQin 

Yeah, `console.error` would be good.
I want it doesn't stop to evaluate.